client/styles/components/_loader.scss

```diff
@@ -0,0 +1,46 @@
+$color_1: #000000;
+
+@keyframes load1 {
+	0%,80%,100% {
+		box-shadow: 0 0;
+		height: 4rem;
+	}
+	40% {
+		box-shadow: 0 -2rem;
+		height: 5rem;
+	}
+}
+.loader {
+	background: #000000;
+	animation: load1 1s infinite ease-in-out;
+	width: 1rem;
```

Comment:
when using `rem`s, calculate them by doing `#{16 / $base-font-size}rem`, so that it's easy to still see what the pixel measurement would be. this is for readability.

Reply:
Should I put them as a comment alongside?